resources/elastic_lb.rb

```diff
@@ -1,9 +1,17 @@
 actions :register, :deregister
 default_action :register
 state_attrs :aws_access_key,
-            :name
+            :elb_name
+
+identity_attr :elb_name
 
 attribute :aws_access_key,        kind_of: String
 attribute :aws_secret_access_key, kind_of: String
 attribute :aws_session_token,     kind_of: String, default: nil
-attribute :name,                  kind_of: String
+attribute :elb_name,              kind_of: String, name_attribute: true
```

Comment:
This should stay as name to align with the naming pattern we generally use / see in chef custom resources